extensions/vscode-api-tests/package.json

```diff
@@ -9,6 +9,7 @@
     "authSession",
     "chatParticipantPrivate",
     "chatProvider",
+    "conditionallyExecuteCommand",
```

Comment:
The linter is throwing an error here, but it was not obvious how I should fix it